src/main/java/io/github/thebusybiscuit/sensibletoolbox/items/EnderLeash.java

```diff
@@ -179,8 +187,9 @@ private YamlConfiguration freezeEntity(Animals target) {
         conf.set("ageLock", target.getAgeLock());
         conf.set("name", target.getCustomName() == null ? "" : target.getCustomName());
         conf.set("nameVisible", target.isCustomNameVisible());
-        conf.set("maxHealth", target.getMaxHealth());
+        conf.set("maxHealth", target.getAttribute(Attribute.GENERIC_MAX_HEALTH).getValue());
         conf.set("health", target.getHealth());
+        conf.set("capture_time", System.currentTimeMillis());
```

Comment:
Nitpicky but all other keys seem to follow the camelCase standard, so this one snake_case exception here is a bit bothersome XD